lib/src/signed_video_h26x_auth.c

```diff
@@ -1028,6 +1028,25 @@ reregister_nalus(signed_video_t *self)
   return status;
 }
 
+static void
+validate_golden_sei(signed_video_t *self, h26x_nalu_list_t *nalu_list)
+{
+  // Check the status of the verified signature hash for the GOP info.
+  switch (self->gop_info->verified_signature_hash) {
+    case 1:
+      nalu_list->first_item->validation_status = '.';
+      break;
+    case 0:
+      nalu_list->first_item->validation_status = 'N';
+      self->latest_validation->authenticity = SV_AUTH_RESULT_NOT_OK;
```

Comment:
I think you should add a TODO here that this result will be overwritten in maybe_validate_gop, so we can fix it in a near future.